Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Define image base arg
-ARG IMAGE_VERSION="ubuntu:20.04"
+ARG IMAGE_VERSION="ubuntu:22.04"
```

Comment:
Updating the Ubuntu version enabled the build to no longer fail when running `scripts/build.sh`.  
Previously, the C++ library build was failing. I replicated @HalosGhost's changes to the Dockerfiles and CMakeLists file in a fresh Ubuntu 22.04 container with forked clone of the repo.  
After installing the necessary build tools/packages, I was able to call the setup dependencies script and run the build successfully.